.github/workflows/pr.yml

```
@@ -1,32 +1,19 @@
 name: PR
 
 on:
-  pull_request:
-    types: [opened, synchronize]
-    branches:
-      - main
+  push: # TODO: revert after resolving merge conflicts
```

```suggestion
  push:
```

did we do the revert?

---

No, sorry, I'll be updating the CI. Since there were significant changes I had commented out for testing.